README.md

```diff
@@ -113,13 +113,24 @@ the vita frontend displays the data collected by `:vita` plus some forms for app
 
 ## install from source
 
-1. create a blank `%vita` desk.
-2. copy in basic app dependencies.
+1. ```bash
+   rm -rI vita-full/
+   find vita -type f | while read f; do { d=$(dirname "$f" | sed "s/^vita/vita-full/"); mkdir -p "$d"; ln -sr -t "$d" "$f"; }; done
+   git clone -b 411k --depth 1 https://github.com/urbit/urbit.git urb
+   cp urb/pkg/arvo/lib/{agentio*,verb*,strand*} vita-full/lib/
+   cp urb/pkg/arvo/sur/{spider*,verb*} vita-full/sur/
+   git clone -b sl/server-schooner-z412k --depth 1 https://github.com/sidnym-ladrut/yard.git yar
```

Comment:
Quick note here: I need to follow up with @sigilante to get the 412k schooner fix into `%yard` proper and then this code can just pull from the master branch. Alternatively, I can just omit these more explicit install instructions.